src/application/components/Application.js

```diff
@@ -83,22 +83,22 @@ export class Application extends Component {
       const modelName = this.props.params.model;
       if (utils.isEnum(modelName, service, importedServices)) {
         // FIXME - not sure if this is enum or model? Need a test case
-        const enumModel = utils.getEnum(modelName, service, importedServices);
-        /* if (enumModel) {
-         *   enumModel.fields = enumModel.values.map(value => (
-         *     { name: value.name, description: value.description, type: 'string', required: false }
-         *   ));
-         * }*/
+        const enumModel: any = utils.getEnum(modelName, service, importedServices);
```

Comment:
bypassing flow check for now as theres not a straightforward way to fix this, but we need the enum to render